code/modules/vore/fluffstuff/custom_items_vr.dm

```diff
@@ -1821,3 +1821,9 @@
 
 	icon_override = 'icons/vore/custom_clothes_vr.dmi'
 	item_state = "blindshades_mob"
+
+//Storesund97 - Aurora
+/obj/item/clothing/accessory/solgov/department/security
+	name = "Old security insignia"
+	desc = "Insignia denoting assignment to the security department. These fit Expeditionary Corps uniforms. This one seems to be from the 2100s..."
+	color = "#bf0000"
```

Comment:
color = "#bf0000" can be deleted, since it inherits it from the parent item /obj/item/clothing/accessory/solgov/department/security